lte/gateway/configs/mme.yml

```diff
@@ -27,8 +27,10 @@ enable_apn_correction: false
 apn_correction_map_list:
         - imsi_prefix: "00101"
           apn_override: "magma.ipv4"
+
+congestion_control_enabled: true
```

Comment:
do we add this here or to gateway.mconfig? I thought the convention is to use mme.yml for configs not streamed down and /etc/magma/gateway.mconfig for the configs streamed down.

Reply:
It is really dependent on the config, I was following the same pattern as we do for `nat_enabled` (give preference to local service config and if not use mconfig value), but thinking about it, local service config not existing will almost never happen. I will update it to only use mconfig